db/migrate/20220112213712_create_messages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMessages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :messages do |t|
+      t.integer :user_id, null: false
```

Comment:
This needs to be configured as a `foreign key`:
```suggestion
      t.integer :user, null: false, foreign_key: true
```